ServerCore/Pages/Puzzles/BulkFiles.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@page  "/{eventId}/{eventRole}/Puzzles/BulkFiles/{handler?}"
+@model ServerCore.Pages.Puzzles.BulkFilesModel
+
+@{
+    ViewData["Title"] = "Bulk File management";
+    ViewData["AdminRoute"] = "../Puzzles/BulkFiles";
+    ViewData["AuthorRoute"] = "../Puzzles/BulkFiles";
+    ViewData["PlayRoute"] = "/Puzzles/Play";
+}
+
+<script type="text/javascript">
+</script>
+
+<h2>@Model.Event.Name: Bulk Files</h2>
+
+<p>
+    <a asp-page="/Puzzles/Index">Back to puzzle list</a>
+</p>
+
+<div>
+    <hr />
+    <h3>What the Bulk File Uploader WILL do</h3>
+    <ul>
+        <li>Upload new versions of files for puzzles/solutions, when stored in folders that match the folder names used previously</li>
```

Comment:
[nit]: "Upload and override" so it lets people know that the these will replace the old versions

Reply:
Clarifying this thx